pkg/dispatcher/dispatcher.go

```diff
@@ -25,25 +25,24 @@ package dispatcher
 import (
 	"fmt"
 	"github.com/onosproject/onos-config/pkg/events"
-	"github.com/onosproject/onos-config/pkg/southbound/topocache"
 	log "k8s.io/klog"
 )
 
 // Dispatcher manages SB and NB configuration event listeners
 type Dispatcher struct {
-	deviceListeners        map[topocache.ID]chan events.ConfigEvent
```

Comment:
I kind of liked using topocache.ID - I know it was only an alias for string, but it was a useful reassurance from the compiler that the right type was being used. In my opinion it improved readability

Reply:
Oh I agree. It’s nice to have. But since the `Device`s are Protobuf generated structs, wed have to provide for the conversion between a type that supports `ID` and the Protobuf generated type. That will introduce some overhead to get the type we want. We could still wrap the `string` `Id` in `ID` when it’s used outside `Device`, but that may affect readability as well by making the two equal values appear different:

Reply:
I think I actually found a way to get the Protobuf generator to generate the `ID` type for Go code. I will experiment with it and see if we can keep it.